File: app/src/main/java/com/example/gaijinsmash/transitapp/fragment/PhoneLinesFragment.java
package com.example.gaijinsmash.transitapp.fragment;

import android.app.Fragment;
import android.app.FragmentManager;
import android.os.Bundle;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;

import com.example.gaijinsmash.transitapp.R;

public class PhoneLinesFragment extends Fragment {
    private View mInflatedView;

    @Override
    public View onCreateView(LayoutInflater inflater, ViewGroup container,
                             Bundle savedInstanceState) {
        mInflatedView = inflater.inflate(R.layout.phone_lines_view, container, false);
        return mInflatedView;
    }

    @Override
    public void onViewCreated(View view, Bundle savedInstanceState) {

    }

    @Override
    public void onPause() {
        super.onPause();
        FragmentManager fm = getFragmentManager();
        fm.popBackStack();
    }
}

File: app/src/main/java/com/zuk0/gaijinsmash/riderz/ui/activity/main/fragment/phone_lines/PhoneLinesFragment.kt
package com.zuk0.gaijinsmash.riderz.ui.activity.main.fragment.phone_lines

import android.content.Intent
import android.net.Uri
import android.os.Bundle
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import androidx.fragment.app.Fragment
import androidx.lifecycle.ViewModelProvider
import androidx.lifecycle.ViewModelProviders
import com.google.android.material.appbar.AppBarLayout
import com.zuk0.gaijinsmash.riderz.R
import com.zuk0.gaijinsmash.riderz.databinding.FragmentPhoneLinesBinding
import com.zuk0.gaijinsmash.riderz.ui.activity.main.fragment.BaseFragment
import javax.inject.Inject

class PhoneLinesFragment : BaseFragment() {

    private lateinit var binding: FragmentPhoneLinesBinding

    @Inject
    lateinit var vmFactory: ViewModelProvider.Factory
    private lateinit var vm: PhoneLinesViewModel

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        vm = ViewModelProvider(this, vmFactory).get(PhoneLinesViewModel::class.java)
    }

    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?,
                              savedInstanceState: Bundle?): View? {
        binding = FragmentPhoneLinesBinding.inflate(inflater)
        return binding.root
    }

    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
        initButtons()
        super.collapseAppBar(activity)
        super.setTitle(activity, getString(R.string.phoneLines_title))
    }

    private fun initButtons() {
        binding.bartPoliceButton.setOnClickListener {
            val number = getString(R.string.phoneLines_police_number)
            val intent = Intent(Intent.ACTION_DIAL, Uri.parse("tel:${vm.formatTelephoneNumber(number)}"))
            startActivity(intent)
        }
        binding.bartComplaintsButton.setOnClickListener {
            val number = getString(R.string.phoneLines_bartComplaints_number)
            val intent = Intent(Intent.ACTION_DIAL, Uri.parse("tel:${vm.formatTelephoneNumber(number)}"))
            startActivity(intent)
        }
        binding.lostAndFoundButton.setOnClickListener {
            val number = getString(R.string.phoneLines_bartLostFound_number)
            val intent = Intent(Intent.ACTION_DIAL, Uri.parse("tel:${vm.formatTelephoneNumber(number)}"))
            startActivity(intent)
        }
    }
}